cmd/dkafka/cdc.go

```diff
@@ -256,12 +256,13 @@ func executeCdC(cmd *cobra.Command, args []string,
 		Irreversible: viper.GetBool("cdc-cmd-irreversible"),
 		Executed:     viper.GetBool("cdc-cmd-executed"),
 
-		Codec:             viper.GetString("cdc-cmd-codec"),
-		SchemaRegistryURL: viper.GetString("cdc-cmd-schema-registry-url"),
-		SchemaNamespace:   viper.GetString("cdc-cmd-namespace"),
-		SchemaVersion:     viper.GetString("cdc-cmd-version"),
-		LocalABIFiles:     localABIFiles,
-		ABICodecGRPCAddr:  viper.GetString("cdc-cmd-abicodec-grpc-addr"),
+		Codec:              viper.GetString("cdc-cmd-codec"),
+		SchemaRegistryURL:  viper.GetString("cdc-cmd-schema-registry-url"),
+		SchemaNamespace:    viper.GetString("cdc-cmd-namespace"),
+		SchemaMajorVersion: viper.GetString("cdc-cmd-major-version"),
+		SchemaVersion:      viper.GetString("cdc-cmd-version"),
+		LocalABIFiles:      localABIFiles,
+		ABICodecGRPCAddr:   viper.GetString("cdc-cmd-abicodec-grpc-addr"),
```

Comment:
Maybe you should use `GetUint` instead of `GetString` as you are expecting a positive number here ;)